xla/tools/replay_computation_bin_test.cc

```diff
@@ -46,13 +47,9 @@ TEST(ReplayComputation, AddHloHost) {
 }
 
 TEST(ReplayComputation, AddHloInterpreter) {
-  // Get relevant paths to run_hlo_module and add.hlo
-  std::string replay_computation_bin = tsl::io::JoinPath(
-      tsl::testing::XlaSrcRoot(), "tools", "replay_computation_interpreter");
```

Comment:
This actually looks like a functional change. One of these is `replay_computation_bin` and the other is `replay_computation_interpreter`. Unless these paths aren't actually used?